### PR TITLE
Staging np

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

**Comment by peterBrxwn** [comment]
[[RN] Create NP version](https://trello.com/c/CjLqASqQ/1883-rn-create-np-version)

**Review by turtledreams — CHANGES_REQUESTED** [review]
well looks like there are some checks we should talk about. call me

**Review by turtledreams — CHANGES_REQUESTED** [review]
there are push related methods you have not touched yet.
there are return types and comments that should be looked at.
this should be a drop in replacement => in the example use the methods we manipulate here so we are sure they wont break when used with np.

File: android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java [hunk 1]
```diff
@@ -874,31 +870,7 @@ public void askForNotificationPermission(ReadableArray args) {
...
-                        log("askForNotificationPermission, Fetching FCM registration token failed", task.getException(), LogLevel.WARNING);
-                        return;
-                    }
-
-                    // Get new FCM registration token
-                    String token = task.getResult();
-                    CountlyPush.onTokenRefresh(token);
-                }
-            });
+            log("askForNotificationPermission, firebaseMessagingInstance is null", LogLevel.WARNING);
```
Comment (ijunaid) [inline android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java:873]: It look like that try block is of now use. we can remove it?

File: android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java [hunk 2]
```diff
@@ -874,31 +870,7 @@ public void askForNotificationPermission(ReadableArray args) {
         }
         CountlyPush.init(configPush);
         try {
-            FirebaseApp.initializeApp(context);
```
Comment (arifBurakDemiray) [inline android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java:877]: we can directly remove this ReactMethod I think

**Review by arifBurakDemiray — CHANGES_REQUESTED** [review]
We can remove all push related calls on the android part I think. However, I am not sure it will be ok

**Review by ijunaid — APPROVED** [review]

File: android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java (hunk 1 above)
Comment (ijunaid) [inline android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java:873]: It look like that try/catch block is of now use. we can remove it?

**Review by ijunaid — CHANGES_REQUESTED** [review]

**Review by arifBurakDemiray — APPROVED** [review]

**Review by ijunaid — APPROVED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**